setup.cfg

```diff
@@ -52,11 +52,13 @@ napari.manifest =
 [options.extras_require]
 dev = 
 	mkdocs
```

Comment:
We'll have to pin to mkdocs version 1.2.4 for the moment if we want mkdocs-gallery to work, see https://github.com/smarie/mkdocs-gallery/issues/57

Reply:
Pinned